insights/parsers/cpuset_cpus.py

```diff
@@ -0,0 +1,51 @@
+"""
+InitProcessCgroup - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
+==============================================================
+
+This parser reads the content of ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
+This file shows the default cgroup cpuset.cpu of system. The format
+of the content is string including comma.
+"""
+
+from .. import parser, CommandParser, LegacyItemAccess
+from insights.specs import Specs
+
+
+@parser(Specs.cpuset_cpus)
+class CpusetCpus(CommandParser, LegacyItemAccess):
+    """
+    Class ``CpusetCpus`` parses the content of the ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
+
+    Attributes:
+
+        cpu_number (int): It is used to display the number of allowed cpu.
```

Comment:
And the `self.data` (or the new name if you will rename to) should be described here,